src/Compilers/VisualBasic/Test/Semantic/Binding/Binder_Expressions_Tests.vb

```diff
@@ -3041,7 +3037,7 @@ Class Program
             Dim x As Object = ""
             Color.M(x)
         Catch e As Exception
-            Console.WriteLine(e)
+            Console.WriteLine(e.GetType())
```

Comment:
If the exception message generated in Roslyn's code base? If not, any test that depends on the exception message belongs in the same code base where the exception is produced.